OWML.Common/MessageType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public enum MessageType
+    {
+        Error,
+        Warning,
+        Message,
```

Comment:
Info?

Reply:
@amazingalek when would we use Info vs Message?

Reply:
Actually nevermind, info would be cool to have too.

Reply:
Added info with blue color